src/main/kotlin/net/axay/hglaborlobby/protection/ServerProtection.kt

```diff
@@ -47,6 +48,7 @@ object ServerProtection {
             if (damager is Player)
                 GeneralProtectionUtils.checkPlayerAction(it, damager)
             else if (damager is Projectile) {
+                if (damager is Firework) it.isCancelled = true
```

Comment:
dont go further if it is a firework (currently performs the "is Player" check aswell)